### PR TITLE
docs: mention about moving docs directory into website

### PR DESCRIPTION
**Pull request by yangshun**
## Motivation

I've been receiving quite a number of questions regarding the Now deployment setup for Docusaurus v1 projects. It'll be helpful to mention that the `docs` have to be moved into `website`.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

See Netlify.

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

File: docs/getting-started-publishing.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ now
 
 **That's all.** Your docs will automatically be deployed.
 
+> Note that the directory structure Now supports is slightly different from the default directory structure of a Docusaurus project - The `docs` directory has to be within the `website` directory, ideally following the directory structure in this example. You will also have to specify a `customDocsPath` value in `siteConfig.js`. Take a look at the the [now-examples repository for a Docusaurus project](https://github.com/zeit/now-examples/tree/master/docusaurus).
```
Comment (wgao19) [inline docs/getting-started-publishing.md]: Somehow "Now" on the first line is capitalized.

**Comment by wgao19** [comment]
i can understand this recommendation. we have moved docs inside website because it somehow didnt work outside and i did not have the work hour to figure that out. 

i would like to also flag that having docs live inside our website isnt optimal for our project because our project is a monorepo, the docsite is one of the repos, but we prefer a doc directory at root

**Comment by yangshun** [comment]
> but we prefer a doc directory at root

Why? What are the use cases for such a structure?

**Comment by wgao19** [comment]
because our docs is about the whole project not individual package, it logically makes more sense to be under root not a specific package. 

also, while the doc site is a nice addition, many of our developers prefer just a link to a markdown file hosted on our git, so we actually prefer them to be outside the docsite

**Pull request merged**